includes/class-notices.php

```diff
@@ -634,7 +634,7 @@ public static function print_frontend_notice( $message, $echo = true, $notice_ty
 		// Note: we will remove give_errors class in future.
 		$error = sprintf(
 			'<div class="give_notices give_errors" id="give_error_%1$s">
-						<p class="give_error give_notice give_%1$s" data-dismissible="%2$s" data-dismiss-interval="%3$d" data-dismiss-type="%4$s">
```

Comment:
@kevinwhoffman The red line in the notice was caused by the hardcoded `give_error` class.

On a success notice, it displayed 3 classes `give_error give_notice give_success`, I think this shouldn't be the case. It should be either `give_error`, `give_success` or `give_warning`. Let me know your views on this, if there can be a possibility of backward compatibility issue.

Reply:
@Sidsector9 I think the `give_error` class needs to stay for now because it also provides the padding and other styles necessary to format the notice in case `give_success` or `give_warning` is not used along with it. I agree it's not ideal. There should be a base class and then modifier classes just for the colors, but we don't want to create rippling effects throughout the add-ons by changing that.

When I tested this notice with Currency Switcher activated, I found that there were styles printed in the `<head>` of the page that were actually causing the red/green mix because they were taking precedent over the stylesheet.

Reply:
Disregard. The red/green was caused by CSS that was pasted in the Customizer's Additional CSS field.